website/versioned_docs/version-0.8.0/flink-quick-start-guide.md

```diff
@@ -5,12 +5,13 @@ toc: true
 last_modified_at: 2020-03-16T11:40:57+08:00
 ---
 
-This guide provides a document at Hudi's capabilities using flink SQL client. Using flink SQL, we can feel the unique charm of using flink stream computing engine to write/read Hudi.
-Reading this guide, you can quickly start using flink hudi, learn configurations, Use data governance methods and study optimization:
+This guide provides a document at Hudi's capabilities using flink SQL. Using flink SQL, we can feel the unique charm of flink stream computing engine on hudi.
+Reading this guide, you can quickly start using flink to write to(read from) hudi, learn configurations and study optimizations:
```

Comment:
Remove `Using flink SQL`.

Reply:
'learn configurations and study optimizations' -> 'Have a deeper understanding of configuration and optimization'